utils/check_dummies.py

```diff
@@ -176,7 +112,8 @@ def create_dummy_files():
     for backend, objects in backend_specific_objects.items():
         backend_name = "[" + ", ".join(f'"{b}"' for b in backend.split("_and_")) + "]"
         dummy_file = "# This file is autogenerated by the command `make fix-copies`, do not edit.\n"
-        dummy_file += "from ..file_utils import requires_backends\n\n"
+        dummy_file += "# flake8: noqa\n"
```

Comment:
This line is necessary since `DummyObject` is sometimes not used in those files and then flake8 is unhappy.